erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -185,6 +185,14 @@ def update_item(obj, target, source_parent):
 
 	return doclist
 
+def set_expired_status():
```

Comment:
@0Pranav Why are you fetching all the quotations and their docs. Just fetch the names of quotations for which you want to update status and update them in one single query. No need to for loop and get doc of each quotation and make explicit commit.